frontend/src/metabase/qb/components/actions/XRaySegment.jsx

```diff
@@ -0,0 +1,24 @@
+/* @flow */
+
+import type {
+    ClickAction,
+    ClickActionProps
+} from "metabase/meta/types/Visualization";
+
+export default ({ card, tableMetadata }: ClickActionProps): ClickAction[] => {
+    console.log(card);
+    if (card.id) {
+        return [
+            {
+                name: "underlying-data",
+                title: "XRay this Card",
+                icon: "table",
+                url: () => {
+                    return "/xray/card/" + card.id;
```

Comment:
We do. There are different fingerprinters for various combinations of fields (eg. DateTime x Number is treated as a time series). This was done a bit to generally and I'm now scaling it back and incorporating it into the card fp